chains/evm/contracts/executor/Executor.sol

```diff
@@ -200,7 +199,7 @@ contract Executor is IExecutor, Ownable2StepMsgSender {
   function getFee(
     uint64 destChainSelector,
     uint16 requestedBlockDepth,
-    Client.CCV[] calldata ccvs,
+    address[] calldata ccvs,
```

Comment:
It never needed the ccv args but it was more efficient to pass in the CCVs struct